scripts/download_elevation_grid.py

```
@@ -0,0 +1,484 @@
+#!/usr/bin/env python3
+
+import asyncio
+import argparse
+import os
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)